content/Guides/ai-gateway.mdx

```diff
@@ -21,6 +21,10 @@ The AI Gateway supports the following providers out of the box:
 - **Mistral**
 - **Perplexity**
 
+<Callout type="info">
+Ready to try out the AI Gateway? [Manage your billing and purchase credits](https://app.agentuity.com/billing) in the Cloud Console.
```

Comment:
I don't really like this wording.  Trying out the AI gateway is automatic kind of, which isn't the same as managing billing.